libiocage/lib/Host.py

```diff
@@ -91,4 +89,4 @@ def processor(self):
 
 class Host(HostGenerator):
 
-    class_distribution = libiocage.lib.Distribution.DistributionGenerator
+    class_distribution = 'libiocage.lib.Distribution.DistributionGenerator'
```

Comment:
it should have been `libiocage.lib.Distribution.Distribution` here. Line 35 should not be deleted as well.

Reply:
n.b.: In `class HostGenerator` this property was `_class_distribution` (yay ☭) but here it's without the leading `_`.